projects/gloo/cli/pkg/cmd/options/options.go

```diff
@@ -47,7 +47,7 @@ type Install struct {
 	Upgrade           bool
 	Namespace         string
 	HelmChartOverride string
-	HelmChartValues   string
+	HelmChartValues   []string
```

Comment:
```suggestion
	HelmChartValuesFiles   []string
```
maybe something like that, a little more explicit?

Reply:
Or `...FileNames` to be consistent with the other fields 🤷‍♂